host/rpi/rpi.go

```diff
@@ -124,28 +120,59 @@ func (d *driver) Init() (bool, error) {
 		return false, errors.New("Raspberry Pi board not detected")
 	}
 
-	// Initialize Version.
+	// Setup headers based on board revision.
 	//
 	// This code is not futureproof, it will error out on a Raspberry Pi 4
 	// whenever it comes out.
+	// Revision codes from: http://elinux.org/RPi_HardwareHistory
+	var has26PinP1Header bool
```

Comment:
Please use the form: `has26PinP1Header := false`

Add has40PinP1Header := false too, since it shouldn't be registered for compute module.